### ISSUE TITLE
TypeError: Cannot read property 'prid' of undefined at getRoom

### ISSUE THREAD

**Issue by ankar84** [RocketChat/Rocket.Chat#17328]
### Description:

Error in logs

### Steps to reproduce:

Just run Rocket Chat for users

### Expected behavior:

Error fixed

### Actual behavior:

From time to time I see that error in Rocket Chat logs

### Server Setup Information:

- Version of Rocket.Chat Server: 3.0.3
- Operating System: CentOS7
- Deployment Method: docker
- Number of Running Instances: 12
- DB Replicaset Oplog: Enabled
- NodeJS Version:  v12.14.0
- MongoDB Version: 4.0.17

### Relevant logs:

```
I20200416-08:51:07.067(0) Exception in callback of async function: TypeError: Cannot read property 'prid' of undefined     at getRoom (server/routes/avatar/room.js:17:11)     at server/routes/avatar/room.js:25:15     at runWithEnvironment (packages/meteor.js:1286:24)     at packages/meteor.js:1299:14     at /app/bundle/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40 
```

**Comment by gbrian** [comment]
😓 wanted to take as my first attempt to jump into Rocket.Chat ...
This seems to be already fixed by @pierre-lehnen-rc 
[FIX] TypeError when trying to load avatar of an invalid room. (#16699) 
https://github.com/RocketChat/Rocket.Chat/commit/2129f5550ca4fae86a7e788746c0020622fcb79f
am I wrong?

**Comment by ankar84** [comment]
> am I wrong?

I think you are right. Because after update to 3.1.1, I didn't see that error.

**Comment by pierre-lehnen-rc** [comment]
Fixed by #16699
